docs/metasploit-framework.wiki/Metasploit-Guide-LDAP.md

```diff
@@ -43,7 +43,7 @@ msf6 auxiliary(gather/ldap_query) > run rhost=192.168.123.13 username=Administra
 [*] Discovering base DN automatically
 [+] 192.168.123.13:389 Discovered base DN: DC=domain,DC=local
 CN=Administrator CN=Users DC=domain DC=local
-==========================================
```

Comment:
Was there a reason for extending this? Did we recently update this? The other examples would also need to be updated if so for consistency's sake.

Reply:
It wasn't aligned before. I think the original was edited but the underline wasn't causing them to be misaligned. If you look at the real output, you'll notice the difference.

Reply:
Fixed this n pushed up changes.